src/bboard/assets/timer_countdown.html

```diff
@@ -0,0 +1,245 @@
+<head>
+  <title>Stop Watch</title>
+  <style>
+    /* Layout and Base Styles */
+    body {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        height: 100vh;
+        margin: 0;
+        background-color: #282c34;
+        font-family: 'Arial', sans-serif;
+        color: #61dafb;
+    }
+    .center-container {
+        position: relative;
+        width: 100%;
+        max-width: 650px;
+        height: 100px;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        cursor: pointer;
+    }
+    h1 {
+        font-size: 5rem;
+        background: rgba(255, 255, 255, 0.1);
+        padding: 20px;
+        border-radius: 10px;
+        box-shadow: 0 4px 20px rgba(0, 0, 0, 0.3);
+        transition: opacity 0.5s ease-in-out;
+        position: absolute;
+        width: 100%;
+        text-align: center;
+    }
+
+    /* Animations */
+    @keyframes shake {
+        0% { transform: rotate(0deg); }
+        25% { transform: rotate(5deg); }
+        50% { transform: rotate(0deg); }
+        75% { transform: rotate(-5deg); }
+        100% { transform: rotate(0deg); }
+    }
+    @keyframes glow {
+        0%, 100% { box-shadow: 0 0 10px #ff4757, 0 0 20px #ff4757, 0 0 30px #ff4757; }
+        50% { box-shadow: 0 0 20px #ff4757, 0 0 30px #ff4757, 0 0 40px #ff4757; }
+    }
+    @keyframes blink {
+        0%, 100% { opacity: 1; }
+        50% { opacity: 0; }
+    }
+
+    /* State Classes */
+    h1.finished {
+        color: #ff4757;
+        animation: shake 0.3s ease-in-out infinite, glow 1s ease-in-out infinite;
+        transform-origin: center;
+    }
+    .fade { transition: opacity 0.5s ease-in-out; }
+    .hidden { opacity: 0; }
+    .visible { opacity: 1; }
+    .blinking-colon { animation: blink 1s step-start infinite; }
+
+    /* Interactive Elements */
+    #startText, #timer {
+        transition: box-shadow 0.3s ease-in-out;
+    }
+    #startText:hover, #timer:hover {
+        box-shadow: 0 0 10px rgba(255, 255, 255, 0.5);
+    }
+    #startText:active, #timer:active {
+        box-shadow: 0 0 20px rgba(97, 218, 251, 0.8);
+    }
+  </style>
+</head>
+<body>
+    <div class="center-container">
+        <h1 id="startText" onclick="startTimer()">Count ⬇ from 10</h1>
+        <h1 id="timer" style="display: none;"></h1>
+    </div>
+
+    <!-- Audio Resources -->
+    <audio id="alarm" src="/assets/sounds/iphone_alarm_sound.mp3"></audio>
+    <audio id="pauseSound" src="/assets/sounds/pause.mp3"></audio>
+    <audio id="resumeSound" src="/assets/sounds/resume.mp3"></audio>
+    <audio id="tickSound" src="/assets/sounds/tick.mp3"></audio>
+
+    <script>
+    /**
+     * Countdown Timer Application
+     * Features:
+     * - 10-second countdown with pause/resume
+     * - Sound effects and visual feedback
+     * - Finish state with animation
+     */
+
+    // Constants and State
+    const INITIAL_SECONDS = 10;
+    const INITIAL_MINUTES = 0;
```

Comment:
This PR is substantially an "add JS code to repo" edit, rather than "add python code". I mean, there _is_ some, but it's mostly JS.

I have some review remarks which are **out-of-scope** for the current PR. Please don't act on them yet. Just keep them in mind for later. You could subsequently create a new Issue + Branch. Nothing I mention is "broken", we're just talking about finessing the details.

- I hacked in a "long HTML string", and am starting to regret it, as I keep seeing subsequent authors copy the style and introduce longer and longer bits of HTML + CSS + JS. A future PR might break out a separate file, a `timer_countdown.js` asset.
- The MM:SS aspect of (0, 10) INITIAL values is interesting. There's no way to take advantage of it ATM, given the ten-second hardcode. We should probably delete the MINUTES aspect, or parameterize "ten seconds" to be something a caller or web client could specify. Having three variables for HH:MM:SS would be silly -- better to phrase it terms of seconds, perhaps thousands of seconds, and worry about formatting HH:MM:SS at the very end.
- The "tick" sound asset seems to be for 1 second, I guess. And several times we re-schedule a replay of it. The ear is sensitive to jitter, and there is a noticeable stutter sometimes. Consider scheduling "next iteration" 100ms before scheduled end-of-play. Alternatively, consider supplying a bigger asset, of 10s duration.
- In keyframes shake, we have 2 "tilted" frames and 3 "normal" frames. This seems an odd asymmetry. Consider going for 50% "tilted" duty cycle.

Summary:

- separate JS asset file
- parameterize the total countdown time (cf Issue 36)
- aesthetic aspects of tick-tock and of shaking

You might choose to tackle Issue #36 first. It will show you how to read numeric parameters from a web client, similar to how a CLI script must crack ARGV parameters.